tokens/finn.no/deprecated.yml

```diff
@@ -530,12 +530,28 @@ s:
         default: blue-600
       positive:
         default: green-50
+        selected:
+          _: green-50
+          hover: green-100
+          active: green-200
```

Comment:
The document you linked to seems to have been updated and those tokens are not needed:
https://docs.google.com/spreadsheets/d/1Q-Tr_dwJVfxgh3527IFjXQKqtnPPM42QeIr-M6q-zu8/edit#gid=888578831&range=68:70